docs/src/components/DocLayout/index.tsx

```diff
@@ -71,120 +73,131 @@ export default function DocLayout({
-                    setTimeout(() => {
-                      // causing Modal to reposition ModalFooter
-                      window.dispatchEvent(new Event("resize"));
-                    }, 70); // not sure why, but this delay is necessary (the exact threshold is 62)
-                  }}
-                />
-              }
-            />
-            <StyledMiddle>
-              <Stack
-                flex
-                direction="column"
-                desktop={{ direction: "row", spacing: "large", align: "stretch" }}
-              >
-                {trail && (
-                  <Hide block on={["smallMobile", "mediumMobile", "largeMobile", "tablet"]}>
-                    <StyledDocNavigationWidth>
-                      <StyledDocNavigationWrapper>
-                        <DocNavigation currentUrl={path} />
-                      </StyledDocNavigationWrapper>
-                    </StyledDocNavigationWidth>
-                  </Hide>
-                )}
-                <StyledMain>
-                  {trail && <Breadcrumbs trail={trail} />}
-                  <Box padding={{ bottom: "XLarge" }}>
-                    <Stack inline align="center" spaceAfter="small">
-                      <AddBookmark />
-                      <Heading as="h1" type="display">
-                        {title}
-                      </Heading>
-                    </Stack>
-                    {description && (
-                      <Box padding={{ left: "XXLarge" }}>
-                        <Text>{description}</Text>
-                      </Box>
-                    )}
-                  </Box>
-                  <StyledMobileOutdent>
-                    {(tabs || headerLink) && (
-                      <Box
-                        display="flex"
-                        align="end"
-                        justify={tabs && tabs.length > 0 ? "between" : "end"}
-                        tablet={{ maxWidth: "80%" }}
-                      >
-                        {tabs && <Tabs activeTab={location.pathname} tabs={tabs} />}
-                        {headerLink && (
-                          // align with tabs
-                          <Box padding={{ bottom: "XXSmall" }}>
-                            <HeaderLink href={headerLink} />
-                          </Box>
-                        )}
-                      </Box>
-                    )}
-                    <Grid columns="1fr" tablet={{ columns: `${tocHasItems ? "80% 20%" : "100%"}` }}>
-                      {tocHasItems && (
-                        <StyledTocWrapper>
-                          <Hide on={["smallMobile", "mediumMobile", "largeMobile"]}>{Toc}</Hide>
-                        </StyledTocWrapper>
+    <>
+      <Helmet>
+        {title && (
+          <title>{getDocumentPageTitle(title, trail ? trail.map(t => t.name) : [])} | Orbit</title>
+        )}
+        {description && <meta name="description" content={description} />}
```

Comment:
I think it'd be good to have a fallback description for Orbit in general.

Reply:
Good point. But we also shouldn't have pages without descriptions, especially not important pages like the home page. I think that was the problem I saw. The solution is to add descriptions in more places.